pkg/scanners/scannerv4/convert.go

```diff
@@ -175,6 +176,7 @@ func vulnerabilities(vulnerabilities map[string]*v4.VulnerabilityReport_Vulnerab
 		if err := setScoresAndScoreVersions(vuln, ccVuln.GetCvssMetrics()); err != nil {
 			utils.Should(err)
 		}
+		setEPSS(vuln, ccVuln.GetEpssMetrics())
```

Comment:
I guess you're following the same pattern for CVSS, but I think it would be better to make this a pure function which returns a `storage.EPSS` object, rather than modifying the `vuln`:

```suggestion
		vuln.Epss = epss(ccVuln.GetEpssMetrics())
```